src/content/en/fundamentals/accessibility/01-focus/focus-nine.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: shared/narrow
```

Comment:
I noticed that none of the initial pages for the chapters have the Next/Previous navigation. It kind of seems like they should have the Next button @Meggin @dgash